### DISCUSSION TITLE
Inverse Filter

### DISCUSSION THREAD

**Discussion by milkas9** [sct/overseerr#3301] [Feature Requests]
I've just upgraded to version 1.32.4 and I am loving the filter options. However I think an inverse option would be a good addition to the filter options. I usually like to browse for movies and series from networks that I have no subscriptions to. 

e.g hide movies or series from netflix, apple tv, etc...

**Comment by KingAndaval** [comment]
Oh my god I just came here to post this exact request!

Ideally I would like to be able to know what I can't stream from my services...

For me, ideal scenario would something like:

**Scenario 1**

1. I maintain a Plex watchlist
2. Only the items that I cannot stream on my services, are automatically requested
3. Overseerr automatically approves requests
4. Items are added to *arr services

**Scenario 2**

1. I maintain a Plex watchlist
2. Plex Watchlist filtered to only show items I have not downloaded, or cannot stream
3. Requests are made within Overseerr, and approved
4. Items are added to *arr services

**Scenario 3**

1. I maintain a Plex watchlist
2. Full watchlist is available to review in Overseerr (currently only 22 items...)
3. When requesting, Overseerr alerts me if an item is on a service I sub to
4. Items are added to *arr services

Currently, auto-requesting and auto approving everyting in my Plex watchlist does not work for me, as about 70% of the watchlist is available in services like Netflix or Prime, that I pay for.

Requesting from my watchlist, in Overseerr, also does not work... as I can only see 22 items :-(

**Comment by Peccadilloz** [comment]
how has this suggestion not more activity? I was really confused as to why this wasn't implemented with the filters in 1.32 as that's what I thought filtering would be for. That's the only reason i still use [mdblist.com](https://mdblist.com/); as you can filter out streaming providers there. 
Would love if this got added to overserr!

**Comment by lostcol0ny** [comment]
I think this is a great suggestion. Having improved filter logic would be really helpful. From what I've been able to tell, also, if I select multiple genres, the results show only movies/series that match all of those genres rather than showing me results that match any of the selected genres, which is a little frustrating.

**Comment by chrisbjr** [comment]
this is a must feature too. i have active subscriptions to netflix and i'd really like to filter out netflix shows/movies that are not there. just maybe an inverse filter would do

**Comment by chroniclex** [comment]
I am bumping this feature request. I would love to see this feature.
